rpcs3/Emu/RSX/CgBinaryProgram.h

```diff
@@ -161,7 +161,7 @@ class CgBinaryDisasm
 	std::string GetMaskDisasm(bool is_sca);
 	std::string GetVecMaskDisasm();
 	std::string GetScaMaskDisasm();
-	std::string GetDSTDisasm(bool is_sca = false);
+	std::string GetDSTDisasm(bool isSca = false);
```

Comment:
since we do snake_case (at least in non GUI code) why did you apply camelCase here instead of fixing it in the implementation?

Reply:
It is the default behavior of Clang-Tidy. I don't mind to change it if you tell me to do so.

Reply:
Remove this one. there are tons of is_sca elsewhere including right above it in GetMaskDisasm() function.